src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
```

Comment:
You can destructure the person to get all the needed props of it
https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0